tests/transaction_tests.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+import datetime
+import os
+import sys
+import struct
+import socket
+import tempfile
+import time
+
+import logging
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+logger = logging.getLogger('program')
+
+import threading
+import subprocess
+
+import inspect
+
+from m3.ice import ICE
+import m3.ice_simulator
+import m3.m3_ice
+
+class TestICE(object):
```

Comment:
IIRC, if you call this class something different (i.e. `class TestTransactionReplay`) it'll make the formatting of the test runner happier